### PR TITLE
[11.0][account_partner_reconcile] splits the button 'Match payments' to 'Match Receivables' and 'Match payables'

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
The previous button was just now working in v11.

### CODE REVIEW

File: account_partner_reconcile/models/res_partner.py [hunk 1]
```diff
@@ -10,13 +10,16 @@ class ResPartner(models.Model):
...
-        # Open reconciliation view for customers
-        accounts = self.env['account.account']
-        accounts += (self.property_account_receivable_id +
-                     self.property_account_payable_id)
+        # Open reconciliation view for customers and suppliers
+        reconcile_mode = self.env.context.get('reconcile_mode', False)
+        if reconcile_mode == 'customers':
+            accounts = self.property_account_receivable_id
+        else:
+            accounts = self.self.property_account_payable_id
```
Comment (MiquelRForgeFlow) [inline account_partner_reconcile/models/res_partner.py]: self.self.

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

File: account_partner_reconcile/models/res_partner.py (hunk 1 above)
Comment (Nikul-Chaudhary) [reply account_partner_reconcile/models/res_partner.py]: It's Simplify, No need to else

```
accounts = self.property_account_payable_id
if reconcile_mode == 'customers':
    accounts = self.property_account_receivable_id
```

**Review by Nikul-Chaudhary — COMMENTED** [review]
Improved Code

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Comment by MiquelRForgeFlow** [comment]
@pedrobaeza ready to merge :green_apple:

**Review by LoisRForgeFlow — APPROVED** [review]

**Pull request merged**